trackintel/preprocessing/staypoints.py

```diff
@@ -270,7 +270,10 @@ def merge_staypoints(staypoints, triplegs, max_time_gap="10min", agg={}):
 
     # a joined dataframe sp_tpls is constructed to add the columns 'type' and 'next_type' to the 'sp_merge' table
     # concat and sort by time
-    sp_tpls = pd.concat([sp_merge, tpls_merge]).sort_values(by=["user_id", "started_at"])
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", message="CRS not set for some of the concatenation inputs.*")
+
+        sp_tpls = pd.concat([sp_merge, tpls_merge]).sort_values(by=["user_id", "started_at"])
```

Comment:
This is related to the issue you opened, right? I think it would be better to have a separate case for the empty tpls case, as you suggested :)

Reply:
Yes! if a df is empty it will be considered as having no crs. setting a crs to an empty df will also cause an error. 

A more natural fix would be separating the cases.

Reply:
So you would add the warning for now, and then we work on the issue in another PR?

Reply:
Yes! a todo comment will be added